go/common/address.go

```diff
@@ -11,12 +11,51 @@ import (
 	"golang.org/x/net/idna"
 )
 
+var (
+	// https://www.iana.org/assignments/iana-ipv4-special-registry/iana-ipv4-special-registry.xhtml
+	reservedRangesIPv4 = []*net.IPNet{
```

Comment:
There's a similar list in https://github.com/oasisprotocol/oasis-core/blob/75f2ac66817647c30b5d7d53fcc88f4418f2424c/go/common/node/address.go#L200-L224

should that one be removed in favor of the added one?

Reply:
Yeah.